fimsauthz-api/fims-authz.yaml

```diff
@@ -337,7 +361,18 @@ components:
             [dss.read.identification_service_areas, rid.read.enhanced_details]
           type: string
           example: "scope1 scope2"
-
+        aud:
+          description: |
+            The "aud" (audience) claim identifies the recipients that the JWT is
+            intended for.  Each principal intended to process the JWT MUST
+            identify itself with a value in the audience claim.  If the principal
+            processing the claim does not identify itself with a value in the
+            "aud" claim when this claim is present, then the JWT MUST be
+            rejected.  "aud" will contain exactly one URL as an identifier of 
+            the intended recipient system.
+          type: string 
+          format: url 
+          example: "https://the-dss.co"
```

Comment:
I recommend "the-dss.co" (hostname only; no protocol)

Reply:
Then we'll need to drop/modify the `format` field.  See above comment.

Reply:
I think the "resource" we're trying to access here (in the terminology of the article) is a USS.  I definitely don't think we want a token per full URL as that would mean a shadow call to the auth server for every USS interaction, defeating many points of the system architecture.  We could have a token per base URL, but that isn't actually a resource at all; it's a set of resources.  As long as we're naming sets of resources, it seems to make sense to move up to hostname as there shouldn't be different authorizations schemes (i.e., different USSs) at the same hostname, and hostnames are unambiguous whereas exactly which form of the base URL (trailing slash or not, etc) increases the space of invalid but somewhat-reasonable requests.  I think the argument could be made that the resource we're trying to access is the USS itself, so the audience should simply be the other USS's ID (the thing in the `sub` claim for its tokens), but that option wasn't chosen in the initial implementation because a hostname can be easily determined from a necessary part of the message (the URL) whereas the USS ID would need to be provided as an additional parameter.

So, yes, I suggest removing `format: url` and just using the hostname.

Reply:
going with `format: hostname` and modifying example to suit.